[#188] Command Bus Infrastructure adjustments (#199)
AxonIQ/reference-guide: extensions/jgroups.md

# JGroups

JGroups is an alternative approach to distributing command bus \(commands\) besides Axon Server.

> The `JGroupsConnector` uses \(as the name already gives away\) [JGroups](http://www.jgroups.org/) as the underlying discovery and dispatching mechanism. Describing the feature set of JGroups is a bit too much for this reference guide, so please refer to the [JGroups User Guide](http://www.jgroups.org/ug.html) for more details.

Reviewer: ```suggestion
The `JGroupsConnector` uses \(as the name already gives away\) [JGroups](http://www.jgroups.org/) as the underlying discovery and dispatching mechanism. Describing the feature set of JGroups is a bit too much for this reference guide, so please refer to the [JGroups User Guide](http://www.jgroups.org/ug.html) for more information.
```

> The `JGroupsConnector` uses \(as the name already gives away\) [JGroups](http://www.jgroups.org/) as the underlying discovery and dispatching mechanism. Describing the feature set of JGroups is a bit too much for this reference guide, so please refer to the [JGroups User Guide](http://www.jgroups.org/ug.html) for more information.

Reviewer: ```suggestion
The `JGroupsConnector` uses \(as the name already gives away\) [JGroups](http://www.jgroups.org/) as the underlying discovery and dispatching mechanism. Describing the features of JGroups is beyond the scope of this reference guide. Please refer to the [JGroups User Guide](http://www.jgroups.org/ug.html) for more information.
```
